internal/bridge/bridge.go

```
@@ -280,25 +284,57 @@ func Spawn(cfg Config) (*Process, error) {
-	cmd.Stdout = io.MultiWriter(os.Stdout, stdoutLog)
+	// Stderr goes directly to both os.Stderr and the log file.
 	cmd.Stderr = io.MultiWriter(os.Stderr, stderrLog)
 
+	// Stdout is piped through the scanner (which tees into the log file and
+	// watches for API/network error markers) instead of being assigned directly
+	// to cmd.Stdout. The scanner pump goroutine is started after cmd.Start().
+	stdoutPipeR, stdoutPipeW := io.Pipe()
+	cmd.Stdout = stdoutPipeW
+
 	if err := cmd.Start(); err != nil {
 		stdinPipe.Close()
```

golangci-lint (2.11.4)

[error] Error return value of `stdinPipe.Close` is not checked

(errcheck)